### PR TITLE
Modified Caesar Cipher test description

### LINKED ISSUE #9842: Caesars Cipher: My code pass all the tests but doesn't validate on FCC
Challenge [Caesars Cipher](https://www.freecodecamp.com/challenges/caesars-cipher#?solution=let%20engine%20%3D%20(v)%20%3D%3E%20%7B%0A%20%20%0A%20%20let%20low%20%3D%20'A'.charCodeAt()%3B%0A%20%20let%20high%20%3D%20'Z'.charCodeAt()%3B%0A%20%20%0A%20%20if%20(!!v.match(%2F%5Ba-z%5D%2Fi))%20%7B%0A%20%20%20%20if%20(v.charCodeAt()%20-%2013%20%3C%20low)%20%7B%0A%20%20%20%20%20%20let%20code%20%3D%20(high%20-%20(low%20-%20(v.charCodeAt()%20-%2013%20%2B%201)))%3B%0A%20%20%20%20%20%20%0A%20%20%20%20%20%20return%20String.fromCharCode(code)%3B%0A%20%20%20%20%7D%0A%20%20%20%20else%20%7B%0A%20%20%20%20%20%20return%20String.fromCharCode(v.charCodeAt()%20-%2013)%3B%0A%20%20%20%20%7D%0A%20%20%7D%0A%20%20else%20%7B%0A%20%20%20%20return%20v%3B%0A%20%20%7D%0A%7D%3B%0A%0Afunction%20rot13(str)%20%7B%20%2F%2F%20LBH%20QVQ%20VG!%0A%20%20%0A%20%20return%20'%22'%20%2B%20str%0A%20%20%20%20.split('')%0A%20%20%20%20.map(engine)%0A%20%20%20%20.join('')%20%2B%20'%22'%3B%0A%7D%0A%0A%2F%2F%20Change%20the%20inputs%20below%20to%20test%0A%2F%2Frot13(%22SERR%20PBQR%20PNZC%22)%3B%0Arot13(%22SERR%20CVMMN!%22)%3B%0A%2F%2Frot13(%22SERR%20YBIR%3F%22)%3B%0A%2F%2Frot13(%22GUR%20DHVPX%20OEBJA%20QBT%20WHZCRQ%20BIRE%20GUR%20YNML%20SBK.%22)%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.106 Safari/537.36</code>.

The code pass all the tests but doesn't validate on FCC editor. Why that? 

My code:
```javascript
let engine = (v) => {
  
  let low = 'A'.charCodeAt();
  let high = 'Z'.charCodeAt();
  
  if (!!v.match(/[a-z]/i)) {
    if (v.charCodeAt() - 13 < low) {
      let code = (high - (low - (v.charCodeAt() - 13 + 1)));
      
      return String.fromCharCode(code);
    }
    else {
      return String.fromCharCode(v.charCodeAt() - 13);
    }
  }
  else {
    return v;
  }
};

function rot13(str) { // LBH QVQ VG!
  
  return '"' + str
    .split('')
    .map(engine)
    .join('') + '"';
}

// Change the inputs below to test
rot13("SERR PBQR PNZC");
rot13("SERR CVMMN!");
rot13("SERR YBIR?");
rot13("GUR DHVPX OEBJA QBT WHZCRQ BIRE GUR YNML SBK.");

```

### PR DESCRIPTION
**Pull request by sahilkhurana19**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #9842
#### Description

Removed quotes from the test case description as discussed.

### CODE REVIEW

**Review by abhisekp — APPROVED** [review]
<h1 align="center"> :sparkles:  :smile_cat:  :+1: LGTM :smiley: :+1:   :sparkles:  </h1>

**Comment by abhisekp** [comment]
@sahilkhurana19 Please rebase your branch against the `staging` branch, resolve the conflicts and push again.

File: seed/challenges/01-front-end-development-certification/basic-bonfires.json [hunk 1]
```diff
@@ -780,10 +780,10 @@
...
-        "assert(rot13(\"GUR DHVPX OEBJA SBK WHZCF BIRE GUR YNML QBT.\") === \"THE QUICK BROWN FOX JUMPS OVER THE LAZY DOG.\", 'message: <code>rot13(\"GUR DHVPX OEBJA SBK WHZCF BIRE GUR YNML QBT.\")</code> should decode to <code>\"THE QUICK BROWN FOX JUMPS OVER THE LAZY DOG.\"</code>');"
+        "assert(rot13(\"SERR PBQR PNZC\") === \"FREE CODE CAMP\", 'message: <code>rot13(\"SERR PBQR PNZC\")</code> should decode to <code>FREE CODE CAMP</code>');",
+        "assert(rot13(\"SERR CVMMN!\") === \"FREE PIZZA!\", 'message: <code>rot13(\"SERR CVMMN!\")</code> should decode to <code>FREE PIZZA!</code>');",
+        "assert(rot13(\"SERR YBIR?\") === \"FREE LOVE?\", 'message: <code>rot13(\"SERR YBIR?\")</code> should decode to <code>FREE LOVE?</code>');",
+        "assert(rot13(\"GUR DHVPX OEBJA QBT WHZCRQ BIRE GUR YNML SBK.\") === \"THE QUICK BROWN DOG JUMPED OVER THE LAZY FOX.\", 'message: <code>rot13(\"GUR DHVPX OEBJA QBT WHZCRQ BIRE GUR YNML SBK.\")</code> should decode to <code>THE QUICK BROWN DOG JUMPED OVER THE LAZY FOX.</code>');"
```
Comment (abhisekp) [inline seed/challenges/01-front-end-development-certification/basic-bonfires.json]: Was this change intentional? i.e. modifying all the test cases?

> THE QUICK BROWN FOX JUMPS OVER THE LAZY DOG.

is actually the correct form.

Comment (sahilkhurana19) [inline seed/challenges/01-front-end-development-certification/basic-bonfires.json]: Thats strange, all i remember was removing the double quotes. I don't know how did that happened. Should I fix that to the correct form?

Comment (sahilkhurana19) [inline seed/challenges/01-front-end-development-certification/basic-bonfires.json]: I had to resolve merge conflicts during rebase so maybe it happened at that time :confused:

Comment (abhisekp) [inline seed/challenges/01-front-end-development-certification/basic-bonfires.json]: Yes please. Revert it as it were in the `staging`.

**Comment by Bouncey** [comment]
Thanks for contributing :+1: 

:tada:
